### PR TITLE
Poppler 0.47.0

### PR DESCRIPTION
**Pull request by edporras**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/poppler.rb [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
   homepage "https://poppler.freedesktop.org/"
-  url "https://poppler.freedesktop.org/poppler-0.46.0.tar.xz"
-  sha256 "967d35d13d61dee2fee656b80efef9e388a9e752bc79b7123f15b49c7769e487"
-  revision 1
-
-  bottle do
-    sha256 "0de1a3dcdf6af3c8dae42c75e19bb8a937e0cc01b0d187ed7d01d198db85b15e" => :el_capitan
-    sha256 "99aa461128d5435261c0a3f6b3f710a67f82a632c07cc3c7f11792419835d58c" => :yosemite
-    sha256 "7d2162da9a64111a7bbc9ace335d78a4eb5ef48e65eb26aa5f21eaec73e40939" => :mavericks
-  end
```
Comment (MikeMcQuaid) [inline Formula/poppler.rb]: Leave this as-is, we'll update it when we pull.

**Comment by edporras** [comment]
Ha.. I guessed wrong. Sorry, Mike. So, should I push a commit to bring it back?

**Comment by MikeMcQuaid** [comment]
Yes please!

File: Formula/poppler.rb [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-  revision 1
+  url "https://poppler.freedesktop.org/poppler-0.47.0.tar.xz"
+  sha256 "b872e7228fc34a71ce4b47a5aea2a57ae67528818fa846e1e0eda089319bd242"
 
   bottle do
     sha256 "0de1a3dcdf6af3c8dae42c75e19bb8a937e0cc01b0d187ed7d01d198db85b15e" => :el_capitan
     sha256 "99aa461128d5435261c0a3f6b3f710a67f82a632c07cc3c7f11792419835d58c" => :yosemite
     sha256 "7d2162da9a64111a7bbc9ace335d78a4eb5ef48e65eb26aa5f21eaec73e40939" => :mavericks
-  end
+  end  option "with-qt", "Build Qt backend"
```
Comment (DomT4) [inline Formula/poppler.rb]: You accidentally moved the `option` here when you put the bottles back 😄. Mind fixing that? Thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/4c42ad659f070d910988e68ac5d4ca91a81d29ce, https://github.com/Homebrew/homebrew-core/commit/812148da58ff3bf193b7a6c0b614b9b305f91969 & https://github.com/Homebrew/homebrew-core/commit/06441ca028eb3eda185e8f707949315a86af9d7c. Thank you once again for your contributions to Homebrew @edporras; always appreciated 🙇.

**Pull request closed without merging**